scripts/data/v2/west-bank-daily.ts

```diff
@@ -10,7 +10,6 @@ const jsonFileName = "west_bank_daily.json";
 
 const columnFilter = new Set([
   "report_date",
-  "report_period",
```

Comment:
this was added on a commit to main to the wrong script

gaza's script does no explicit column / field selection so it's not needed